script.js

```diff
@@ -0,0 +1,177 @@
+// 1.) Go through the items and find all results that have kind of shopping#product. Print the count of these results. Where else is this count information stored in the search results?
```

Comment:
wow....

Reply:
Sorry, I left it a wow and didn't elaborate. You're the only one that gave many solutions to a problem. I'm impressed. 

Along the same line, I'll recommend you look into another way of looping.
See if you can do it with a forEach method.

Reply:
not sure I understand, isn't my options 1 already using a `.forEach()`?

Reply:
Awesome, sorry didn't really read very in depth